### DISCUSSION TITLE
A suggestion for how to make Beat Cop better.

### DISCUSSION THREAD

**Discussion by trevormoney1** [FakeFishGames/Barotrauma#6874] [Unstable feedback]
Currently, the primary ability, the tackle, is quite bad. In my experience testing it over and over, it’s basically impossible to cuff someone using it unless you happen to be the terminator or a professional speedrunner, and it is not effective in a fight, as the act of running at someone and ragdolling is practically suicidal, will get you hit with a stun baton, knife, or shot with whatever firearm they feel, and is the most unsafe option. The only part of Beat Cop worth it is the 30% more stun, but even that isn’t enough to make the beat cop line as good as Munitions Expertise, which, even ignoring the new ammo type to craft, effectively allows you to craft 50% more ammo for the same price, including stun darts, which pairs well with the fulgurium stun gun darts of the next choice. Whilst more stun seems good with fulgurium darts, it isn’t as effective as getting more darts out of it, unless you’re pairing with another security officer to craft the darts instead. Regardless, the Beat Cop skill seems mostly inferior from a practical standpoint, and if buffed to give more stun, may be too overpowered, as it may result in making subduing a target far too easy. I suggest replacing the tackling effect entirely with a new crafting item to rival the explosive shells one skill over, and will pair quite nicely with the 30% more stun without directly being affected by it.

**The Riot Shield**
_We learned a thing or two from our Earth ancestors, this shield is one of those lessons._

The riot shield will use some similar functionality to the mudraptor shell currently in the game. It will be one-handed, but too large to fit in the standard inventory. Like the assault rifle, it can be equipped in the belt slot to be back equipped. The height of the shield will be enough to cover the head, and leaves the legs exposed unless crouching. When an attack, ranged or melee, connects with the shield, it’s hurtbox is negated and cannot damage the user on the other side of the shield; The shield loses durability when this happens, proportional to the damage it negates. Since the shield is one-handed, the player can wield smaller firearms such as pistols, stun-guns, and most melee weapons in their other hand, allowing them to fight back. When the user of the shield aims a weapon, melee or ranged, or is swinging a weapon, the hitbox of the shield goes away until the attack finishes, to leave an opening for counter attack. If the user of the shield plays their cards right, this could be used to help waste ammunition from pirates during a boarding process, making capture far easier to finish with a shield and stun baton. This would compliment it’s stun increase as it would make the shield a good tool for capture, which is the main purpose of stun, outside of stunlocking during a fight. It would also add a new mechanic to melee combat that would increase the skill level of melee combatants, and I believe this would be much better received than the current tackle Beat Cop possesses, as most people I’ve inquired about this skill have very negative views of it both in balance and mechanics. Hopefully this keeps to the themes of Beat Cop and will add to the expanding arsenal of the security team.

**Comment by StruggleDuck** [comment]
I agree that beat cop needs to be adjusted a bit, as it is a tad bit unreliable and clunky, but I also feel that you misunderstand what beatcop is meant to do. I feel as though beatcop is supposed to primarily used to tackle a rouge crew member who is lightly armed, or to quickly take down an assailant. This would make it so that you wouldn't necessarily need to carry around a bunch of stun items, freeing up inventory space, and also allowing you save your resources for more dire situations.

**Comment by Githubberclown** [comment]
I want to see this in game, i approve

**Reply by StruggleDuck** [reply]
I agree, having some kind of shield in game would be cool to see

**Comment by doornydew** [comment]
maybe make tackling someone stun them for just a bit longer, but allow access to their inventory during the tackle? (so if you ARE quick enough and prepared you can hotkey the cuffs over and put them on them)
